pkg/util/export/etl/csv.go

```diff
@@ -58,13 +61,21 @@ func (w *CSVWriter) GetContent() string {
 }
 
 func (w *CSVWriter) FlushAndClose() (int, error) {
-	n, err := w.writer.WriteString(w.buf.String())
-	if err != nil {
-		return 0, err
+	// First write rows to sqlWriter, if not working, write to csv file
```

Comment:
Where is the csv file stored and what is the role of these csv files?

Reply:
CSV files are stored in S3. They were designed for external tables. Now, they only act as a backup when can not insert into the internal table.